__tests__/app.test.js

```diff
@@ -156,3 +124,54 @@ describe("\nGET /api/reviews/:review_id/comments\n", () => {
       });
   });
 });
+describe("\nPOST /api/reviews/:review_id/comments\n", () => {
+  test("get 201 status and add new review comment to database and return new comment", () => {
+    const review_id = 2;
+    const newComment = {
+      review_id: review_id,
+      username: "dav3rid",
+      body: "This is exactly what I was saying to my cat last night.",
+    };
+    return request(app)
+      .post(`/api/reviews/${review_id}/comments`)
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        expect(response.body).toMatchObject({
+          body: expect.any(String),
+        });
+      });
+  });
+  test("get 404 'not found' when review_id not found", () => {
+    const review_id = 2000;
+    return request(app)
+      .get(`/api/reviews/${review_id}`)
+      .expect(404)
+      .then((response) => expect(response.body.msg).toBe("not found"));
+  });
+  test("get 400 'invalid input' when body not provided", () => {
+    const review_id = 2;
+    const newComment = {
+      review_id: review_id,
+      username: "dav3rid",
+    };
+    return request(app)
+      .post(`/api/reviews/${review_id}/comments`)
+      .send(newComment)
+      .expect(400)
+      .then((response) => expect(response.body.msg).toBe("invalid input"));
+  });
+  test("get 404 'user not found' when username doesn't exist in users", () => {
+    const review_id = 2;
+    const newComment = {
+      review_id: review_id,
+      username: "Justine",
+      body: "This is exactly what I was saying to my cat last night.",
+    };
+    return request(app)
+      .post(`/api/reviews/${review_id}/comments`)
+      .send(newComment)
+      .expect(404)
+      .then((response) => expect(response.body.msg).toBe("user not found"));
+  });
+});
```

Comment:
Just missing a few more tests here. What happens when username isn't on the request body? What happens if the request body has other, unnecessary keys?